core/math/vector3.h

```diff
@@ -290,6 +291,14 @@ Vector3 Vector3::project(const Vector3 &p_to) const {
 	return p_to * (dot(p_to) / p_to.length_squared());
 }
 
+real_t Vector3::cos_to(const Vector3 &p_to) const {
+	if (is_zero_approx() || p_to.is_zero_approx()) {
+		return 0; // for consistency with normalized().dot(p_to.normalized())
```

Comment:
Please follow this guideline for formatting comments https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide

Reply:
Done.